modules/geo-layers/src/mvt-tile-layer/mvt-tile-layer.js

```diff
@@ -0,0 +1,67 @@
+import {Matrix4} from 'math.gl';
+import {MVTLoader} from '@loaders.gl/mvt';
+import {load} from '@loaders.gl/core';
+import {COORDINATE_SYSTEM} from '@deck.gl/core';
+import {GeoJsonLayer} from '@deck.gl/layers';
+
+import TileLayer from '../tile-layer/tile-layer';
+
+const defaultProps = Object.assign({}, TileLayer.defaultProps, {
+  renderSubLayers: {type: 'function', value: renderSubLayers, compare: false},
+  urlTemplates: []
```

Comment:
Start a documentation page in `docs/layers`?

Reply:
Good point!